tensorflow/core/platform/default/BUILD

```diff
@@ -269,6 +269,8 @@ cc_library(
         "//tensorflow/core/platform:mem.h",
         "//tensorflow/core/platform:numa.h",
         "//tensorflow/core/platform:snappy.h",
```

Comment:
Please sort the list alphabetically:
        "//tensorflow/core/platform:numa.h",
        "//tensorflow/core/platform:profile_utils/cpu_utils.h",
        "//tensorflow/core/platform:profile_utils/i_cpu_utils_helper.h",
        "//tensorflow/core/platform:snappy.h",

Reply:
I will do that, thanks!